natives.json

```diff
@@ -14580,7 +14580,7 @@
 			"build": "323"
 		},
 		"0xCE6294A232D03786": {
-			"name": "_GET_ENTITY_BONE_ROTATION",
+			"name": "_GET_WORLD_ROTATION_OF_ENTITY_BONE",

```

Comment:
This PR will never be merged if you continue to synchronize the many mistakes (or in this particular case: an outdated name) of the FiveM native repo with this repository. ... 

Break it up into more manageable/maintainable pieces.

Reply:
Its hard to judge whether a name is outdated or not as many sources have different names and as long as its not confirmed, the name describing what the native does the most is a good name I guess.

Splitting so many changes into multiple PRs isn't something I am planning to do, but anybody else is free to do it